cmd/defaults_windows.go

```diff
@@ -0,0 +1,17 @@
+package cmd
+
+const (
+	DefaultLayersDir           = `c:\layers`
```

Comment:
Shouldn't we rely on acceptance tests (right now, they're in `pack`'s repo) to catch such bugs?

Reply:
Right now, we do not acceptance test all of the flags and defaults for every phase. While we are working improving the test coverage I doubt we will ever get to 100% b/c high-level tests are costly in terms of speed and maintenance.

Reply:
I see. For now there is some coverage via pack tests. Would you be open to seeing how it goes, and if it proves to be a problem, address it then? OS-specific files seem like the pattern for situations like this, looking at the standard golang libraries, and I struggle with adding complexity to the implementation when it feels like there's a better solution to the "keep in sync" problem lurking somewhere. Thoughts?

Reply:
You could argue that a file named `defaults_unix.go` would warrant a look a the `defaults_windows.go` file which will sit next to it in most IDEs, if making any large changes.

Reply:
I am not sure `pack` acceptance tests provide any coverage here b/c `pack` tends to set all of these vars explicitly rather than relying on the defaults. Using some combination of `filepath.Join` plus an OS specific root dir and executable extension feels much safer to me, and also doesn't seem like that much work? This is the kind of thing that has historically caused issues.

Reply:
OK, will do :)

Reply:
@ekcasey OK, so I took a middle-of-the-road approach and only used separate os-specific files for the parts that are not likely to ever change (the system root and executable extension string). The rest is provided through read-only values (functions) that use `filepath.Join`. :)

Reply:
I'll mark this (now outdated) thread as resolved, but feel free to comment on the new code if needed.